.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 Cargo.lock
+.idea
```

Comment:
Let's not add editor-specific stuff here. I'd suggest using a global gitignore on your device instead.

Reply:
TIL about global gitignore. On it.